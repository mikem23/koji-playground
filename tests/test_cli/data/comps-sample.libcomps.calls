[call.groupListAdd('tag', u'additional-devel', biarchonly=True, description=u'Additional development headers and libraries for developing applications', display_name=u'Additional Development', force=None, is_default=False, langonly=u'fr', uservisible=False),
 call.groupPackageListAdd('tag', u'additional-devel', u'alsa-lib-devel', basearchonly=False, force=None, type='default'),
 call.groupPackageListAdd('tag', u'additional-devel', u'audit-libs-devel', basearchonly=False, force=None, type='default'),
 call.groupPackageListAdd('tag', u'additional-devel', u'binutils-devel', basearchonly=False, force=None, type='default'),
 call.groupPackageListAdd('tag', u'additional-devel', u'boost-devel', basearchonly=False, force=None, type='default'),
 call.groupPackageListAdd('tag', u'additional-devel', u'bzip2-devel', basearchonly=False, force=None, type='default'),
 call.groupPackageListAdd('tag', u'additional-devel', u'cyrus-sasl-devel', basearchonly=False, force=None, type='default'),
 call.groupListAdd('tag', u'backup-client', biarchonly=False, description=u'Client tools for connecting to a backup server and doing backups.', display_name=u'Backup Client', force=None, is_default=True, langonly=None, uservisible=True),
 call.groupPackageListAdd('tag', u'backup-client', u'amanda-client', basearchonly=False, force=None, type='mandatory'),
 call.groupPackageListAdd('tag', u'backup-client', u'bacula-client', basearchonly=False, force=None, type='optional'),
 call.groupListAdd('tag', u'backup-server', biarchonly=False, description=u"Software to centralize your infrastructure's backups.", display_name=u'Backup Server', force=None, is_default=False, langonly=None, uservisible=True),
 call.groupPackageListAdd('tag', u'backup-server', u'amanda-server', basearchonly=False, force=None, type='mandatory'),
 call.groupPackageListAdd('tag', u'backup-server', u'mt-st', basearchonly=False, force=None, type='optional'),
 call.groupPackageListAdd('tag', u'backup-server', u'mtx', basearchonly=False, force=None, type='optional'),
 call.groupListAdd('tag', u'ansible-node', biarchonly=False, description=None, display_name=u'Ansible node', force=None, is_default=False, langonly=None, uservisible=True),
 call.groupPackageListAdd('tag', u'ansible-node', u'python2-dnf', basearchonly=False, force=None, type='mandatory'),
 call.groupPackageListAdd('tag', u'ansible-node', u'libselinux-python', basearchonly=False, force=None, requires=u'selinux-policy', type='conditional'),
 call.groupListAdd('tag', u'd-development', biarchonly=False, description=u'These include development tools and libraries such as ldc, and geany-tag.', display_name=u'D Development Tools and Libraries', force=None, is_default=False, langonly=None, uservisible=True),
 call.groupPackageListAdd('tag', u'd-development', u'ldc', basearchonly=True, force=None, type='mandatory'),
 call.groupPackageListAdd('tag', u'd-development', u'ldc-druntime', basearchonly=True, force=None, type='mandatory'),
 call.groupPackageListAdd('tag', u'd-development', u'ldc-druntime-devel', basearchonly=True, force=None, type='mandatory'),
 call.groupPackageListAdd('tag', u'd-development', u'ldc-phobos-devel', basearchonly=True, force=None, type='mandatory'),
 call.groupPackageListAdd('tag', u'd-development', u'make', basearchonly=False, force=None, type='mandatory'),
 call.groupPackageListAdd('tag', u'd-development', u'pkgconfig', basearchonly=False, force=None, type='mandatory'),
 call.groupPackageListAdd('tag', u'd-development', u'ctags', basearchonly=False, force=None, type='default'),
 call.groupPackageListAdd('tag', u'd-development', u'indent', basearchonly=False, force=None, type='default'),
 call.groupPackageListAdd('tag', u'd-development', u'astyle', basearchonly=False, force=None, type='optional'),
 call.groupPackageListAdd('tag', u'd-development', u'cmake', basearchonly=False, force=None, type='optional'),
 call.groupPackageListAdd('tag', u'd-development', u'derelict-devel', basearchonly=False, force=None, type='optional'),
 call.groupPackageListAdd('tag', u'd-development', u'geany', basearchonly=False, force=None, type='optional'),
 call.groupPackageListAdd('tag', u'd-development', u'gl3n-devel', basearchonly=False, force=None, type='optional'),
 call.groupPackageListAdd('tag', u'd-development', u'insight', basearchonly=False, force=None, type='optional'),
 call.groupPackageListAdd('tag', u'd-development', u'nemiver', basearchonly=False, force=None, type='optional'),
 call.groupPackageListAdd('tag', u'd-development', u'uncrustify', basearchonly=False, force=None, type='optional'),
 call.groupListAdd('tag', u'empty-group-1', biarchonly=False, description=u'empty group 1 desc', display_name=u'empty group 1', force=None, is_default=False, langonly=None, uservisible=True),
 call.groupListAdd('tag', u'empty-group-2', biarchonly=False, description=u'empty group 2 desc', display_name=u'empty group 2', force=None, is_default=False, langonly=None, uservisible=True),
 call.groupListAdd('tag', u'unknown-group', biarchonly=False, description=u'unknown group desc', display_name=u'unknown group', force=None, is_default=False, langonly=None, uservisible=True),
 call.groupPackageListAdd('tag', u'unknown-group', u'unknown', basearchonly=False, force=None, type='unknown'),
 call.groupPackageListAdd('tag', u'unknown-group', u'unknown2', basearchonly=False, force=None, type='unknown')]
