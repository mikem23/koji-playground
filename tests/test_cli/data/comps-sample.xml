<?xml version='1.0' encoding='UTF-8'?>
<!DOCTYPE comps PUBLIC "-//Red Hat, Inc.//DTD Comps info//EN" "comps.dtd">
<comps>
  <group>
    <id>additional-devel</id>
    <name>Additional Development</name>
    <description>Additional development headers and libraries for developing applications</description>
    <default>false</default>
    <uservisible>false</uservisible>
    <biarchonly>true</biarchonly>
    <langonly>fr</langonly>
    <packagelist>
      <packagereq type="default">alsa-lib-devel</packagereq>
      <packagereq type="default">audit-libs-devel</packagereq>
      <packagereq type="default">binutils-devel</packagereq>
      <packagereq type="default">boost-devel</packagereq>
      <packagereq type="default">bzip2-devel</packagereq>
      <packagereq type="default">cyrus-sasl-devel</packagereq>
    </packagelist>
  </group>
  <group>
    <id>backup-client</id>
    <name>Backup Client</name>
    <description>Client tools for connecting to a backup server and doing backups.</description>
    <default>true</default>
    <uservisible>true</uservisible>
    <packagelist>
      <packagereq type="mandatory">amanda-client</packagereq>
      <packagereq type="optional">bacula-client</packagereq>
    </packagelist>
  </group>
  <group>
    <id>backup-server</id>
    <name>Backup Server</name>
    <description>Software to centralize your infrastructure's backups.</description>
    <default>false</default>
    <uservisible>true</uservisible>
    <packagelist>
      <packagereq type="mandatory">amanda-server</packagereq>
      <packagereq type="optional">mt-st</packagereq>
      <packagereq type="optional">mtx</packagereq>
    </packagelist>
  </group>
  <group>
    <id>ansible-node</id>
    <name>Ansible node</name>
    <default>false</default>
    <uservisible>true</uservisible>
    <packagelist>
      <packagereq type="mandatory">python2-dnf</packagereq>
      <packagereq type="conditional" requires="selinux-policy">libselinux-python</packagereq>
    </packagelist>
  </group>
  <group>
    <id>d-development</id>
    <name>D Development Tools and Libraries</name>
    <description>These include development tools and libraries such as ldc, and geany-tag.</description>
    <default>false</default>
    <uservisible>true</uservisible>
    <packagelist>
      <packagereq type="mandatory" basearchonly="true">ldc</packagereq>
      <packagereq type="mandatory" basearchonly="true">ldc-druntime</packagereq>
      <packagereq type="mandatory" basearchonly="true">ldc-druntime-devel</packagereq>
      <packagereq type="mandatory" basearchonly="true">ldc-phobos-devel</packagereq>
      <packagereq type="mandatory">make</packagereq>
      <packagereq type="mandatory">pkgconfig</packagereq>
      <packagereq type="default">ctags</packagereq>
      <packagereq type="default">indent</packagereq>
      <packagereq type="optional">astyle</packagereq>
      <packagereq type="optional">cmake</packagereq>
      <packagereq type="optional">derelict-devel</packagereq>
      <packagereq type="optional">geany</packagereq>
      <packagereq type="optional">gl3n-devel</packagereq>
      <packagereq type="optional">insight</packagereq>
      <packagereq type="optional">nemiver</packagereq>
      <packagereq type="optional">uncrustify</packagereq>
    </packagelist>
  </group>
  <group>
    <id>empty-group-1</id>
    <name>empty group 1</name>
    <description>empty group 1 desc</description>
    <default>false</default>
    <uservisible>true</uservisible>
    <packagelist/>
  </group>
  <group>
    <id>empty-group-2</id>
    <name>empty group 2</name>
    <description>empty group 2 desc</description>
    <default>false</default>
    <uservisible>true</uservisible>
  </group>
    <group>
    <id>unknown-group</id>
    <name>unknown group</name>
    <description>unknown group desc</description>
    <default>false</default>
    <uservisible>true</uservisible>
    <packagelist>
      <packagereq type="unknown">unknown</packagereq>
      <packagereq type="what">unknown2</packagereq>
    </packagelist>
  </group>
</comps>
