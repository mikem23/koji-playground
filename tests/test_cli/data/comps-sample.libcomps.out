Group: additional-devel (Additional Development)
  Package: alsa-lib-devel: {'type': 'default', 'basearchonly': False}
  Package: audit-libs-devel: {'type': 'default', 'basearchonly': False}
  Package: binutils-devel: {'type': 'default', 'basearchonly': False}
  Package: boost-devel: {'type': 'default', 'basearchonly': False}
  Package: bzip2-devel: {'type': 'default', 'basearchonly': False}
  Package: cyrus-sasl-devel: {'type': 'default', 'basearchonly': False}
Group: backup-client (Backup Client)
  Package: amanda-client: {'type': 'mandatory', 'basearchonly': False}
  Package: bacula-client: {'type': 'optional', 'basearchonly': False}
Group: backup-server (Backup Server)
  Package: amanda-server: {'type': 'mandatory', 'basearchonly': False}
  Package: mt-st: {'type': 'optional', 'basearchonly': False}
  Package: mtx: {'type': 'optional', 'basearchonly': False}
Group: ansible-node (Ansible node)
  Package: python2-dnf: {'type': 'mandatory', 'basearchonly': False}
  Package: libselinux-python: {'requires': u'selinux-policy', 'type': 'conditional', 'basearchonly': False}
Group: d-development (D Development Tools and Libraries)
  Package: ldc: {'type': 'mandatory', 'basearchonly': True}
  Package: ldc-druntime: {'type': 'mandatory', 'basearchonly': True}
  Package: ldc-druntime-devel: {'type': 'mandatory', 'basearchonly': True}
  Package: ldc-phobos-devel: {'type': 'mandatory', 'basearchonly': True}
  Package: make: {'type': 'mandatory', 'basearchonly': False}
  Package: pkgconfig: {'type': 'mandatory', 'basearchonly': False}
  Package: ctags: {'type': 'default', 'basearchonly': False}
  Package: indent: {'type': 'default', 'basearchonly': False}
  Package: astyle: {'type': 'optional', 'basearchonly': False}
  Package: cmake: {'type': 'optional', 'basearchonly': False}
  Package: derelict-devel: {'type': 'optional', 'basearchonly': False}
  Package: geany: {'type': 'optional', 'basearchonly': False}
  Package: gl3n-devel: {'type': 'optional', 'basearchonly': False}
  Package: insight: {'type': 'optional', 'basearchonly': False}
  Package: nemiver: {'type': 'optional', 'basearchonly': False}
  Package: uncrustify: {'type': 'optional', 'basearchonly': False}
Group: empty-group-1 (empty group 1)
Group: empty-group-2 (empty group 2)
Group: unknown-group (unknown group)
  Package: unknown: {'type': 'unknown', 'basearchonly': False}
  Package: unknown2: {'type': 'unknown', 'basearchonly': False}
